forge/db/controllers/AccessToken.js

```diff
@@ -409,5 +409,35 @@ module.exports = {
             ownerType: 'broker'
         })
         return { token }
+    },
+
+    createTokenForNPM: async function (app, entity, team, scope = ['team:packages:read']) {
+        // might want to prefix the entityId with `p-`, `d-` and `u-` rather than relying on
```

Comment:
> might want to prefix the entityId with `p-`, `d-` and `u-` rather than relying on

That does feel sensible to me.

Reply:
@knolleary I've pushed an update with this.

Shout if you want to run the whole stack and I can help getting verdaccio and the plugin running